tools/build_config/build.ps1

```diff
@@ -36,6 +36,10 @@ param (
   [switch][Alias("p")]$package,
   # Print the versions of libraries being used e.g. Matlab.
   [switch][Alias("v")]$print_versions,
+  # Build docs
+  [switch][Alias("d")]$docs,
+  # Push docs to github
+  [switch]$push_docs,
```

Comment:
Is not  having an alias for push deliberate to make it unlikely to happen by accident?

Reply:
Deliberate, sensible choices would have been `-p` or `-d` and those would lead too easily to typos, and users shouldn't be doing it anyway.